File: app/src/main/java/com/apps/wow/droider/player/MainView.java
package com.apps.wow.droider.player;


import android.widget.ImageButton;

/**
 * Created by Jackson on 06/01/2017.
 */

public interface MainView {

    boolean isControlActivated();

    void setIsControlActivated(boolean isActivated);

    void setControlButtonImageResource(int resource);

    void setVisibilityToControlButton(int visibility);

    ImageButton getControlButton();

    void refreshNotification();

    void showToast(String text);

    String getPodcastTitle();
}

File: app/src/main/java/com/apps/wow/droider/Player/MainView.kt
package com.apps.wow.droider.Player


import android.widget.ImageButton

/**
 * Created by Jackson on 06/01/2017.
 */

interface MainView {

    var isControlActivated: Boolean

    fun setControlButtonImageResource(resource: Int)

    fun setVisibilityToControlButton(visibility: Int)

    val controlButton: ImageButton

    fun refreshNotification()

    fun showToast(text: String)

    val podcastTitle: String

    fun setupSeekbar()
}
